### PR TITLE
fix/safe-file-size

### PR DESCRIPTION
**Pull request by FeliceGeracitano**
- [x] test

### CODE REVIEW

**Comment by rbelling** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:tada: 0</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 94478d7dcd253b6d4d50a99199b408742fcfa1c4
</p>

**Comment by rbelling** [comment]
hi, can we hold on merging this one until the table UI is ready? thanks!

**Comment by FeliceGeracitano** [comment]
@rbelling sure

File: src/commands/compare.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const exec = util.promisify(childProcessExec);
 
 export default class Compare extends Command {
   public static description = 'Compare JS/CSS bundles of two git branches.';
-  public static examples = [`$ npx bundle-checker compare --targetBranch='master`];
+  public static examples = [`$ npx bundle-checker compare`];
```
Comment (FeliceGeracitano) [inline src/commands/compare.ts]: lets assume master by default as well

File: src/commands/compare.ts [hunk 2]
```diff
@@ -31,6 +36,6 @@ export default class Compare extends Command {
       const { stdout } = await exec('git rev-parse --abbrev-ref HEAD');
       defaults.currentBranch = stdout.trim();
     }
-    return { ...flags, ...defaults };
```
Comment (FeliceGeracitano) [inline src/commands/compare.ts]: defaults need to be overridden and not ovveride

**Pull request merged**